assets/src/domain/eventEditor/ui/datetimes/datesList/index.tsx

```diff
@@ -31,4 +30,12 @@ const DatesList: React.FC = () => {
 	);
 };
 
-export default DatesList;
+const WrappedDatesList: React.FC = () => {
+	return (
+		<DatetimesListProvider>
+			<DatesList />
+		</DatetimesListProvider>
+	);
+};
+
+export default WrappedDatesList;
```

Comment:
👍 

I still think HoCs are useful in some cases

although technically i guess this really isn't an HoC 🤷‍♂

Reply:
I don't think bad about HOCs. I used similar approach in TAM form content context

![image](https://user-images.githubusercontent.com/18226415/76200755-2b18bc80-6218-11ea-85e1-54464fa17862.png)